### DISCUSSION TITLE
Disable vhost, but still use port 80 ?

### DISCUSSION THREAD

**Discussion by cburkins** [dokku/dokku#4846] [Q&A]
Is it possible to disable vhost capability, but still serve on port 80, redirecting to the port of the backend app ?

I'm just doing a bit of testing, and don't want to setup a domain.    So I want to serve on http://192.168.1.66:80, and have that redirect to port 3000 (where I'm running a simple express hello world app).

I can share my config if that's helpful....    I can get this working on port 81, but not port 80.  On port 80, I continue to get the generic Nginx welcome page.

**Comment by josegonzalez** [comment]
You may wish to delete the default nginx conf. I believe if multiple things match, nginx will just take the first one. Thus, without a domain, you can only have 1 app listen to a given port at a time.
